README.md

```diff
@@ -84,6 +84,23 @@ number of production topologies.
 See the collection of [docs](docs/README.md) for in depth guides on how use
 Kubernetes Network Emulation (KNE).
 
+## Disclaimers
+
+### Usage Metrics Reporting
+
+We collect anonymous usage reporting metrics to gauge the health and
```

Comment:
As an initial sentence this may raise blood pressure.  Maybe something like:

The KNE CLI optionally collects anonymous usage reporting metrics. **This defaults to off.**
We use the metrics to gauge the health and ...